guides/plugins/plugins/storefront/add-scss-variables-via-subscriber.md

```diff
@@ -0,0 +1,238 @@
+# Add SCSS variables
+
```

Comment:
If this only applies from a specific SW version, please add the version hint here.

Reply:
This already applies to older versions, it was just moved because it is no longer the standard.